### ISSUE TITLE
[Docs] Troubleshooting Model Generation

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#11476]
### Current State
The process of generating Meshery models can fail for any number of reasons.

### Desired State
A new section in the [Contributing to Models](https://docs.meshery.io/project/contributing/contributing-models) page (or a new page) for troubleshooting model generation, highlighting common model generation failures and suggested fixes for those failures

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by vaishnaviadhau** [comment]
@leecalcote I would like to work on this issue, pls assign this to me.

**Comment by ENUMERA8OR** [comment]
Can we start contributing to this feature ? @leecalcote

**Comment by leecalcote** [comment]
Yes, thanks much for jumping in and volunteering. Here's what help guarantee both your success and assignment of the issue (and there are plenty more where this issue comes from):

1. Confirm that you're able to build the project, specifically, the components necessary to resolve this issue.
1. Confirm the approach you intend to take to resolve the issue.

**Comment by Shridhar2104** [comment]
Hey Lee, I would like to work on this issue!
I have setup docs locally and I have mesheryctl installed, I hope this satisfies the requirements to work on the issue.
My approach would be listing the following issues - 
1. Invalid Spreadsheet ID
2. Invalid Spreadsheet Credentials
3. Invalid Schema 
I would like to add those few in the trouble shooting guide firstly.

**Comment by Jougan-0** [comment]
@Shridhar2104 good going. Along with that there are various reasons why the models itself fails and not the process specifically. You can refer to [Meshery Integration's](https://docs.google.com/spreadsheets/d/1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw/edit?gid=1048915141#gid=1048915141) Model Generator Error sheet for all the different errors that can occur.
If you are stuck anywhere feel free to ping me.

**Comment by Shridhar2104** [comment]
Thankyou for giving some direction @Jougan-0 , I will look into it and let you know

**Comment by TomiwaAribisala-git** [comment]
Hi @Shridhar2104 are you still working on this issue or I can take it?

**Comment by Shridhar2104** [comment]
I'm working on it @TomiwaAribisala-git

**Comment by Shridhar2104** [comment]
[https://docs.google.com/spreadsheets/d/1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw/edit?gid=1048915141#gid=1048915141](Meshery Integration's) in the sheet above for the first error, no package on artifact-hub, we can create create a new command in CLI tool using Cobra that checks for the availability of a package on Artifact Hub. If the package is not found, it provides troubleshooting steps like - 
Verify the package name is correct
Ensure you have internet access.
Visit https://artifacthub.io/ and search for the package manually.

 Any changes I need to make for the above approach?
